content/en/docs/tasks/tools/included/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Tools Included"
+description: "Pages and snippets to be included in the main install-kubectl.md page."
+toc_hide: true
+---
```

Comment:
This looks like it might be intended to work like a Hugo [headless bundle](https://gohugo.io/content-management/page-bundles/#headless-bundle); in which case, does:
```suggestion
---
title: "Tools Included"
description: "Pages and snippets to be included in the main install-kubectl.md page."
headless: true
---
```
work?